File: Go/GORM/models/Person.go
package models

import (
	"fmt"
	"time"

	"github.com/google/uuid"
	"gorm.io/gorm"
)

type Person struct {
	Id        string `gorm:"primarykey"`
	Name      string
	Address   string
	BirthDate time.Time
	Employees []Employee `gorm:"constraint:OnDelete:CASCADE;"`
}

func (Person) TableName() string {
	return "people"
}

func (p Person) String() string {
	return fmt.Sprintf(
		"Person { Id: %s, Name: %s, Address: %s, BirthDate: %s }",
		p.Id,
		p.Name,
		p.Address,
		p.BirthDate.Format("02.01.2006"),
	)
}

func (p *Person) BeforeCreate(_ *gorm.DB) error {
	p.Id = uuid.NewString()
	return nil
}

File: Python/Peewee/models/person.py
from datetime import date
from uuid import uuid4

from models.base_model import BaseModel

from peewee import CharField, DateField


class Person(BaseModel):
    id: str = CharField(primary_key=True, default=uuid4)
    name: str = CharField()
    address: str = CharField()
    birth_date: date = DateField(formats=DateField.formats[0])

    class Meta:
        table_name = 'people'
        model_name = 'Person'

    def __str__(self):
        return (f'{{ Person: {{ Id: {self.id}, Name: {self.name}, '
                f'Address: {self.address}, '
                f'BirthDate: {self.birth_date} }} }}')
